specs/phase1/beacon-chain.md

```diff
@@ -761,7 +799,9 @@ def process_crosslink_for_shard(state: BeaconState,
 
         # Attestation <-> shard transition consistency
         assert shard_transition_root == hash_tree_root(shard_transition)
-        assert attestation.data.head_shard_root == shard_transition.shard_data_roots[-1]
+        assert attestation.data.head_shard_root == shard_transition.shard_data_roots[
```

Comment:
I think `attestation` is out of scope here. It's outside of `for attestation in transition_attestations:`
Should it be `attestations[0]`? Assuming same shard transition root implies having the same shard head root

Reply:
@terencechain Nice catch!

I think it should be inside `for attestation in transition_attestations` here? Updated in https://github.com/ethereum/eth2.0-specs/pull/1703/commits/e758fb76c2ee67afa63a581c5b03022c6a739875

Reply:
looks good